pep-0649.rst

```diff
@@ -1,11 +1,12 @@
 PEP: 649
 Title: Deferred Evaluation Of Annotations Using Descriptors
 Author: Larry Hastings <larry@hastings.org>
-Status: Draft
+Status: Accepted
 Type: Standards Track
 Topic: Typing
 Content-Type: text/x-rst
 Created: 11-Jan-2021
+Python-Version: 3.13
 Post-History: 11-Jan-2021, 11-Apr-2021, 19-Apr-2023
```

Comment:
```suggestion
Post-History: 11-Jan-2021, 11-Apr-2021, 19-Apr-2023
Resolution: https://discuss.python.org/t/a-massive-pep-649-update-with-some-major-course-corrections/25672/62
```

(https://peps.python.org/pep-0001/#pep-header-preamble)

Reply:
Good call!  FWIW I'm actually gonna go with [the other post from the SC announcing they were accepting 649.](https://discuss.python.org/t/pep-649-deferred-evaluation-of-annotations-tentatively-accepted/21331/43)  This was their own thread about acceptance of 649 so it seemed more on point somehow.